CharlieBackend.Api/Validators/AccountDTOValidators/ChangeCurrentPasswordDtoValidator.cs

```diff
@@ -16,11 +13,15 @@ public ChangeCurrentPasswordDtoValidator()
                 .EmailAddress().WithMessage("Incorrect email")
                 .MaximumLength(50).WithMessage("Email cannot be greateh than {MaxLength} symbols");
```

Comment:
I think FluentValidation package provides the default error messages that would be pretty same to these.
Please, remove unnecessary error messages

Reply:
It have already done in https://github.com/ita-social-projects/WhatBackend/pull/437